crypto/encoding/codec_test.go

```diff
@@ -58,7 +58,7 @@ func TestPubKeyToFromProto(t *testing.T) {
 	// unsupported key type
 	_, err = PubKeyToProto(unsupportedPubKey{})
 	require.Error(t, err)
-	assert.Equal(t, ErrUnsupportedKey{KeyType: "encoding.unsupportedPubKey"}, err)
+	assert.Equal(t, ErrUnsupportedKey{KeyType: "unsupportedPubKey"}, err)
```

Comment:
```suggestion
	assert.Equal(t, ErrUnsupportedKey{KeyType: unsupportedPubKey{}.Type()}, err)
```
would this be better ?